uber/models/department.py

```diff
@@ -33,7 +33,11 @@
     MagModel.metadata,
     Column('dept_membership_id', UUID, ForeignKey('dept_membership.id')),
     Column('dept_role_id', UUID, ForeignKey('dept_role.id')),
-    UniqueConstraint('dept_membership_id', 'dept_role_id'))
+    UniqueConstraint('dept_membership_id', 'dept_role_id'),
+    Index('ix_dept_membership_dept_role_dept_role_id', 'dept_role_id'),
```

Comment:
Er, wait, what does `ix` stand for? Did you tell me already and I forgot? :E

Reply:
index